cmd/serverless-init/initcontainer/initcontainer.go

```diff
@@ -43,6 +44,11 @@ func Run(cloudService cloudservice.CloudService, logConfig *serverlessLog.Config
 
 func execute(cloudService cloudservice.CloudService, config *serverlessLog.Config, metricAgent *metrics.ServerlessMetricAgent, traceAgent *trace.ServerlessTraceAgent, args []string) error {
 	commandName, commandArgs := buildCommandParam(args)
+
+	// Add our tracer settings
+	fs := afero.NewOsFs()
+	AutoInstrumentTracer(fs)
```

Comment:
nit: If the `fs` is only ever going to be used in `AutoInstrumentTracer`, then I'd think it'd be better to define the `fs` inside of the `AutoInstrumentTracer` method instead of here.

Reply:
Oh I see, you're doing it this way for testing purposes. 👍🏽

Reply:
Yep! It looks like we're using afero like this in other places. Not the best to have code specific for tests but tbh I think in this case it's worth it.